paper-badge.html

```diff
@@ -79,7 +87,14 @@
       }
     </style>
 
-    <div id="badge">{{label}}</div>
+    <div id="badge">
```

Comment:
The badge element was actually there before I modified the code. I think it needs to be there becausethe host element needs to be a positioned block element in order for the position updates to be applied correctly, but because the badge content needs to be centered it's inside a wrapping flexbox element (the `#badge` el).